java-maven/README.md

```diff
@@ -10,14 +10,15 @@ This sample may be out of date. If you face any issues, please reach out to mail
 * [Using docker compose](#using-docker-compose)
 * [Rookout Integration explained](#rookout-integration-explained)
 ## Running locally
-1. Run the Rookout agent:
-    ``` bash
-    $ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
-    ```
-2. Compile the project jar and download the Java agent:
+1. Clone and compile the project jar and download the Java agent:
      ```bash
+    $ git clone https://github.com/Rookout/deployment-examples/tree/master/java-maven
     $ make build
     ```
+2. Export organization token
+     ```bash
+     export ROOKOUT_TOKEN=<Your-Token>
+     ```
 3. Run:
     ```bash
     $ make run
```

Comment:
@ManuelNemirovsky no more makes :(

Reply:
it was one of the old java-maven files
my mistake